agent/ssm/ssm.go

```diff
@@ -43,6 +43,25 @@ func GetSecretsFromSSM(names []string, client SSMClient) (map[string]string, err
 	return extractSSMValues(out)
 }
 
+func GetParametersFromSSM(names []string, client SSMClient) (map[string]string, error) {
```

Comment:
Can you add a todo here: **TODO: Refactor GetSecretsFromSSM function so it can be reused here.**. The function should be refactored but this is not something you need to do.